fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeManager.java

```diff
@@ -277,16 +279,12 @@ private void initPrivilegeCollectionAllObjects(
 
     // called by initBuiltinRolesAndUsers()
     private RolePrivilegeCollection initBuiltinRoleUnlocked(long roleId, String name) {
-        if (!roleNameToId.containsKey(name)) {
-            RolePrivilegeCollection collection = new RolePrivilegeCollection(
-                    name, RolePrivilegeCollection.RoleFlags.MUTABLE);
-            roleIdToPrivilegeCollection.put(roleId, collection);
-            roleNameToId.put(name, roleId);
-            LOG.info("create built-in role {}[{}]", name, roleId);
-            return collection;
-        }
-        // public roles may be changed and persisted in image before restarted
-        return null;
+        RolePrivilegeCollection collection = new RolePrivilegeCollection(
+                name, RolePrivilegeCollection.RoleFlags.MUTABLE);
+        roleIdToPrivilegeCollection.put(roleId, collection);
+        roleNameToId.put(name, roleId);
+        LOG.info("create built-in role {}[{}]", name, roleId);
+        return collection;
```

Comment:
And could you write some details about this bug in the pr description and add some UTs?